endpoints/openrtb2/test_utils.go

```diff
@@ -1086,12 +1089,35 @@ func (tc *testConfigValues) getAdaptersConfigMap() map[string]config.Adapter {
 	return adaptersConfig
 }
 
+// exchangeTestWrapper is a wrapper that asserts the openrtb2 bid request just before the HoldAuction call
```

Comment:
In the presence of `"expectedValidatedBidRequest"` in a given JSON test, this wrapper will assert this field right before passing the bidRequest to `exchange.HoldAuction()`